docs/UserGuide.md

```diff
@@ -112,7 +116,7 @@ Here are some commands that you might find it handy when using TR4CKER.
 
 ### 3.1.1. Viewing help : `help`
 If you cannot recall the command you need, you can enter the help command in any panel.
-A help window will pop out with a link to this User Guide.
+You will be switched to help tab and a help window will pop out with a link to this User Guide.
```

Comment:
phrasing abit awkward, maybe can say `TR4CKER will switch to help tab and a help window will pop out with a link to this User Guide.`?

Reply:
Is it even a tab? Something like a drop down bar?